### ISSUE TITLE
suggest conda list --only-pip

### ISSUE THREAD

**Issue by m0ast** [conda/conda#8286]
Is there someway for conda list to only show pip packages. (--only-pip)
As `pip list` show me conda packages too, it's actually difficult to only list the ones i installed with pip.

**Comment by kylekeppler** [comment]
Conda env export has a section for pip packages that probably wouldn't be hard to parse.

**Comment by soapy1** [comment]
you can use grep to get pip packages
```
conda list | grep pypi
```
